mriqc/workflows/functional/base.py

```diff
@@ -684,3 +715,45 @@ def _parse_tout(in_file):
 
     data = np.loadtxt(in_file)  # pylint: disable=no-member
     return data.mean()
+
+
+def _apply_transforms(in_file, in_xfm):
+    from pathlib import Path
+    from nitransforms.io.afni import AFNILinearTransformArray
+    from mriqc.utils.bids import derive_bids_fname
+
+    realigned = AFNILinearTransformArray.from_filename(
+        in_xfm,
+        reference=in_file,
+    ).apply(in_file)
+
+    out_file = derive_bids_fname(
+        in_file,
+        entity="desc-realigned",
+        newpath=Path.cwd(),
+        absolute=True,
+    )
+
+    realigned.to_filename(out_file)
+    return str(out_file)
+
+
+def select_echo2(in_files):
```

Comment:
Why 2?

Reply:
To get the ball rolling. In the end, we will probably look at the TEs and pick the one closest to the typical ~30ms of SE-EPI.